flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/common/CommonLookupJoin.scala

```diff
@@ -17,13 +17,27 @@
  */
 package org.apache.flink.table.planner.plan.nodes.common
 
+import java.util.Collections
+import java.util.concurrent.CompletableFuture
+
+import com.google.common.primitives.Primitives
+import org.apache.calcite.plan.{RelOptCluster, RelTraitSet}
+import org.apache.calcite.rel.`type`.{RelDataType, RelDataTypeField}
+import org.apache.calcite.rel.core.{JoinInfo, JoinRelType}
+import org.apache.calcite.rel.{RelNode, RelWriter, SingleRel}
+import org.apache.calcite.rex._
+import org.apache.calcite.sql.SqlKind
+import org.apache.calcite.sql.fun.SqlStdOperatorTable
+import org.apache.calcite.sql.validate.SqlValidatorUtil
+import org.apache.calcite.tools.RelBuilder
+import org.apache.calcite.util.mapping.IntPair
```

Comment:
Nit: afaik, the project imports convention is to list
 1. Flink imports first;
 2. other imports (non `java` and `javax`);
 3. `javax`;
 4. `java`.

At least for `*.java` source files, there should be a code check failure if imports are in wrong order.